tfjs-backend-wasm/scripts/build-benchmark.sh

```diff
@@ -1,7 +1,7 @@
 yarn build
 yarn rollup -c
 
-cp dist/tf-backend-wasm.js ../e2e/benchmarks/
-cp wasm-out/tfjs-backend-wasm.wasm ../e2e/benchmarks/
-cp wasm-out/tfjs-backend-wasm-simd.wasm ../e2e/benchmarks/
-cp wasm-out/tfjs-backend-wasm-threaded-simd.wasm ../e2e/benchmarks/
+cp dist/tf-backend-wasm.js ../e2e/benchmarks/local-benchmark/
+cp wasm-out/tfjs-backend-wasm.wasm ../e2e/benchmarks/local-benchmark/
+cp wasm-out/tfjs-backend-wasm-simd.wasm ../e2e/benchmarks/local-benchmark/
+cp wasm-out/tfjs-backend-wasm-threaded-simd.wasm ../e2e/benchmarks/local-benchmark/
```

Comment:
Hi @annxingyuan , do you think I should make this change? because I would migrate the benchmark webpage tool from `tfjs/e2e/benchmarks/` to `tfjs/e2e/benchmarks/local-benchmark`.

Reply:
sure that looks good to me thanks lin!

Reply:
Thanks Ann!